basic_hashtable/b_hashtables.c

```diff
@@ -94,7 +109,17 @@ void hash_table_insert(BasicHashTable *ht, char *key, char *value)
  ****/
 void hash_table_remove(BasicHashTable *ht, char *key)
 {
-
+  int h = hash(key, ht->capacity);
+  if (ht->storage[h] != NULL)
+  {
+    destroy_pair(ht->storage[h]);
+    ht->storage[h] = NULL;
+  }
+  else
+  {
+    printf("\nCannot remove, index %d does not exist\n", h);
+    exit(1);
```

Comment:
in "real life" this program would be a part of a toolchain and the programming just exiting is not the desired behavior.  Have `return;` instead of `exit(1);`